block/src/Pos/Block/Network/Logic.hs

```diff
@@ -289,7 +289,7 @@ applyWithoutRollback pm diffusion blocks = do
         :: HeaderHash -> m (HeaderHash, Either ApplyBlocksException HeaderHash)
     applyWithoutRollbackDo curTip = do
         logInfo "Verifying and applying blocks..."
-        res <- verifyAndApplyBlocks pm False blocks
+        res <- fmap fst <$> verifyAndApplyBlocks pm False blocks
```

Comment:
In the custom prelude at my last employer we had `fmap2` and the `<$$>` operator which were both defined as `fmap . fmap` :)